Automattic/jetpack
# Add tracking for backup storage info bar chart

#### Changes proposed in this Pull Request:

* This PR adds tracking for the new bar chart added in #23374 

#### Jetpack product discussion
HACK WEEK project p6TEKc-61M-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:

* Boot up the branch and run a JT site and connect Jetpack with Free plan
* Goto At a Glance page. 
* Ensure that you have more than 20 posts published on the site OR you can also manually change (via React Dev tools) the `posts` prop to more than 20 in `BackupUpdare` component added in #23374 
* After the BarChart is visible, confirm that you see in Network tab new tracking request with event name `jetpack_wpa_aag_backup_bar_chart_view`
* Close the popup that blocks the bar chart
* Confirm that you see in Network tab new tracking request with event name being `jetpack_wpa_click`, `target` and `type` being `backup_bar_chart_close` and `dismiss` respectively.
